docs/technical_reference/property_models/water.rst

```diff
@@ -0,0 +1,122 @@
+Water Property Package
+======================
+
+This package implements property relationships for pure water.
+
+This water property package:
+   * supports only H2O
+   * supports liquid and vapor phases
+   * is formulated on a mass basis
+   * does not support dynamics
+   * pressure-dependency of specific enthalpy is incorporated
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Components", ":math:`j`", "['H2O']"
+   "Phases", ":math:`p`", "['Liq', 'Vap']"
+
+State variables
+---------------
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Component mass flowrate", ":math:`M_j`", "flow_mass_phase_comp", "[p, j]", ":math:`\text{kg/s}`"
+   "Temperature", ":math:`T`", "temperature", "None", ":math:`\text{K}`"
+   "Pressure", ":math:`P`", "pressure", "None", ":math:`\text{Pa}`"
+
+Properties
+----------
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Mass density of pure water", ":math:`\rho`", "dens_mass_phase", "[p]", ":math:`\text{kg/}\text{m}^3`"
+   "Phase volumetric flowrate", ":math:`Q_p`", "flow_vol_phase", "[p]", ":math:`\text{m}^3\text{/s}`"
+   "Volumetric flowrate", ":math:`Q`", "flow_vol", "None", ":math:`\text{m}^3\text{/s}`"
+   "Specific enthalpy", ":math:`\widehat{H}`", "enth_mass_phase", "[p]", ":math:`\text{J/kg}`"
+   "Enthalpy flow", ":math:`H`", "enth_flow_phase", "[p]", ":math:`\text{J/s}`"
+   "Saturation pressure", ":math:`P_v`", "pressure_sat", "None", ":math:`\text{Pa}`"
+   "Specific heat capacity", ":math:`c_p`", "cp_mass_phase", "[p]", ":math:`\text{J/kg/K}`"
+   "Latent heat of vaporization", ":math:`h_{vap}`", "dh_vap_mass", "None", ":math:`\text{J/kg}`"
+   "Component mole flowrate", ":math:`N_j`", "flow_mol_phase_comp", "[p, j]", ":math:`\text{mole/s}`"
+   "Component mole fraction", ":math:`y_j`", "mole_frac_phase_comp", "[p, j]", ":math:`\text{dimensionless}`"
+
+Relationships
+-------------
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Component mass fraction", ":math:`x_j = \frac{M_j}{\sum_{j} M_j}`"
+   "Mass density of liquid", "Equation 8 in Sharqawy et al. (2010)"
+   "Mass density of vapor*", ":math:`\rho = \frac{Pm}{nRT}`"
+   "Volumetric flowrate", ":math:`Q = \frac{\sum_{j} M_j}{\rho}`"
+   "Mass concentration", ":math:`C_j = x_j \cdotp \rho`"
+   "Specific enthalpy of liquid", "Equations 25-27 in Nayar et al. (2016)"
+   "Specific enthalpy of vapor", "Equations 25-27 in Nayar et al. (2016) :math:`+ h_{vap}`"
+   "Enthalpy flow", ":math:`H = \sum_{j} M_j \cdotp \widehat{H}`"
+   "Component mole flowrate", ":math:`N_j = \frac{M_j}{MW_j}`"
+   "Component mole fraction", ":math:`y_j = \frac{N_j}{\sum_{j} N_j}`"
+   "Saturation pressure", "Equation 6 in Nayar et al. (2016)"
+   "Specific heat capacity of liquid", "Equation 9 in Sharqawy et al. (2010)"
+   "Specific heat capacity of vapor", "Shomate equation from NIST WebBook"
```

Comment:
I think you should consider adding an entry for the NIST WebBook to the list of references at the end of the page.